meson.build

```diff
@@ -0,0 +1,89 @@
+project('libva-utils', 'c', 'cpp',
+        version : '2.1.1.1',
+        default_options : [
+          'warning_level=2',
+          'c_std=gnu99',
+          'default_library=static',
+        ],
+        license : 'MIT',
+        meson_version : '>= 0.44.0')
```

Comment:
I experienced the following error when running 'meson build' on Ubuntu 17.10

```
Meson encountered an error in file meson.build, line 1, column 0:
Meson version is 0.42.1 but project requires >= 0.44.0.
```
Is it possible to relax the requirement on meson version?

Reply:
Sure, I think I just copied this from another project...
But 0.42+ should be fine.